interface/templates/interface/generics/review_modal.html

```diff
@@ -11,7 +11,14 @@ <h5 class="modal-title" id="reviewLabel">Review submission {{ sub.pk }}</h5>
         <form id="reviewForm" method="post" action="review">
           {% csrf_token %}
           <div class="form-group">
-            <label for="message-text" class="col-form-label">Review details:</label>
+            <div class="row">
+                <div class="col">
+                  <label for="message-text">Review details:</label>
+                </div>
+                <div class="col">
+                  <a href="https://github.com/vmck/acs-interface/wiki/Features#review-submissions" class="float-right" style="text-decoration: none;">Format syntax</a>
```

Comment:
Can you please supply and image so we can see the result better :smile:.

Reply:
![cropped](https://user-images.githubusercontent.com/56245077/110374660-97abde80-8059-11eb-8b59-e7aa151f958d.png)